server/server.js

```diff
@@ -77,7 +78,7 @@ app.use(helmet({
       ],
       styleSrc: ["'self'", "'unsafe-inline'", 'https://fonts.googleapis.com', 'https://fonts.gstatic.com'],
       fontSrc: ["'self'", 'https://*.hotjar.com', 'https://fonts.googleapis.com', 'https://fonts.gstatic.com'],
-      imgSrc: ['https://*.hotjar.com', 'https://*.ndla.no', 'https://www.google-analytics.com', 'https://stats.g.doubleclick.net', 'data: https://i.ytimg.com https://pi.tedcdn.com http://*.ndlap3.seria.net https://*.gstatic.com'],
+      imgSrc: ['https://*.hotjar.com', 'https://*.ndla.no', 'https://www.google-analytics.com', 'https://stats.g.doubleclick.net', 'https://ssl-ndla.tns-cs.net', 'data: https://i.ytimg.com https://pi.tedcdn.com http://*.ndlap3.seria.net https://*.gstatic.com'],
```

Comment:
Kunne vi gjort dette felles for alle frontendene? Ser for meg at de fleste her vil gjelde alle sammen... (cc @oyvinmar )

Reply:
Hvis jeg kan komme med et innspill her:
Dette er noe vi kunne dratt inn i `ndla-util` feks, men det er vel ikke alle som er like over alle platformer? Hva skal skje med dem?

Reply:
Det er vel mulig å lage et delt sett med alle default/felles strengene, med mulighet for legge til ekstra. Altså en funksjon: `configureCSP(custom)`.

Reply:
Foreslår at det gjøres i en egen oppgave. Oppretter en issue på det.